modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-values-factory/src/main/java/com/liferay/dynamic/data/mapping/form/values/factory/internal/DDMFormValuesFactoryImpl.java

```diff
@@ -382,8 +382,8 @@ protected String getDDMFormFieldParameterValue(
 			DDMFormRendererConstants.DDM_FORM_FIELD_LANGUAGE_ID_SEPARATOR);
 		sb.append(LocaleUtil.toLanguageId(locale));
 
-		if (Validator.isNull(
-				ParamUtil.getString(httpServletRequest, sb.toString()))) {
+		if (ParamUtil.getString(httpServletRequest, sb.toString(), null) ==
```

Comment:
Hi, @adolfopa. This will not work for Grid fields since their parameter value is actually an array. This part will now become [this](https://github.com/liferay-data-engine/liferay-portal/pull/236/commits/b67a2b3608e0fbb3ab4c2d171ca9e4c538a94eb4) once that PR passes its review.